controllers/mongodb_users.go

```diff
@@ -0,0 +1,51 @@
+package controllers
+
+import (
+	"fmt"
+
+	mdbv1 "github.com/mongodb/mongodb-kubernetes-operator/api/v1"
+	"github.com/mongodb/mongodb-kubernetes-operator/pkg/kube/secret"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (r ReplicaSetReconciler) ensureUserResources(mdb mdbv1.MongoDBCommunity) error {
+	for _, user := range mdb.GetScramUsers() {
+		secretNamespacedName := types.NamespacedName{Name: user.PasswordSecretName, Namespace: mdb.NamespacedName().Namespace}
```

Comment:
[nit] we can just do `mdb.Namespace` instead of passing through `NamespacedName()`

Reply:
done